covalent/executor/executor_plugins/local.py

```diff
@@ -59,6 +62,21 @@ class LocalExecutor(BaseExecutor):
     Local executor class that directly invokes the input function.
     """
 
+    def __init__(self, workdir: str = "", *args, **kwargs) -> None:
+        if not workdir:
+            try:
+                workdir = get_config("executors.local.workdir")
+            except KeyError:
+                workdir = _EXECUTOR_PLUGIN_DEFAULTS["workdir"]
+                info_msg = f"Couldn't find `executors.local.workdir` in config, creating a default one at {workdir}"
+                app_log.info(info_msg)
+                print(info_msg)
```

Comment:
Printing and log level change here as well.

Reply:
done